api/swagger.yaml

```diff
@@ -766,13 +788,20 @@ components:
 
     Data:
       type: object
+      required:
+        - current-tick
+        - planets
+        - players
+        - ships
       properties:
         current-tick:
           $ref: '#/components/schemas/CurrentTick'
         planets:
           $ref: '#/components/schemas/Planets'
-        player:
-          $ref: '#/components/schemas/PlayerId'
+        player_id:
+          type: integer
+          format: int64
+          nullable: true
```

Comment:
nullable? That's for the website?

Reply:
yes, anyone may call this and get more / less data.